doc/nrf/known_issues.rst

```diff
@@ -882,17 +882,17 @@ tx_buffer_length set incorrectly
 Zephyr
 ******
 
-.. rst-class:: v1-4-0
+.. rst-class:: v1-4-99-dev1 v1-4-0
 
 NCSDK-6330: USB Mass Storage Sample Application fails MSC Tests from USB3CV test tool
   :ref:`zephyr:usb_mass` fails the USB3CV compliance Command Set Test from the MSC Tests suite.
 
-.. rst-class:: v1-4-0
+.. rst-class:: v1-4-99-dev1 v1-4-0
```

Comment:
NCSDK-6328 is fixed upstream/sdk-zephyr.

Reply:
Removed.